.jazzy.yaml

```diff
@@ -0,0 +1,5 @@
+module: MongoSwift
```

Comment:
There were getting to be a lot of arguments in our call to jazzy in `make documentation` so I pulled them out into here.
we also need to manually specify the authors now because Jazzy was previously reading them out from our podspec.